Sources/Sentry/SentryCoreDataTrackingIntegration.m

```diff
@@ -0,0 +1,54 @@
+#import "SentryCoreDataTrackingIntegration.h"
+#import "SentryCoreDataSwizzling.h"
+#import "SentryCoreDataTracker.h"
+#import "SentryLog.h"
+#import "SentryNSDataSwizzling.h"
+#import "SentryOptions+Private.h"
+#import "SentryOptions.h"
+
+@interface
+SentryCoreDataTrackingIntegration ()
+
+@property (nonatomic, strong) SentryCoreDataTracker *tracker;
+
+@end
+
+@implementation SentryCoreDataTrackingIntegration
+
+- (void)installWithOptions:(SentryOptions *)options
+{
```

Comment:
We *always* need to check for `tracesSampleRate or `tracesSampler`. Without it, no transaction is ever sent out, and that's the case for users that don't want performance monitoring.

Do we have an extension method of sorts that checks all requirements? I feel we could pull these checks into a method:

guardFeature(options, requiresSwizzling:true, optionsKey:"enableAutoPerformanceTracking", featureName:"CoreData performance monitoring", whatever other requirements).

Reply:
Agreed.